posthog/api/test/test_organization_feature_flag.py

```diff
@@ -140,6 +145,25 @@ def test_copy_feature_flag_update_existing(self):
             ensure_experience_continuity=False,
         )
 
+        # The following instances must remain linked to the existing flag after overwriting it
+        experiment = Experiment.objects.create(team=self.team_2, created_by=self.user, feature_flag_id=existing_flag.id)
+        survey = Survey.objects.create(team=self.team, created_by=self.user, linked_flag=existing_flag)
+        feature = EarlyAccessFeature.objects.create(
+            team=self.team,
+            feature_flag=existing_flag,
+        )
+        analytics_dashboard = Dashboard.objects.create(
+            team=self.team,
+            created_by=self.user,
+        )
+        existing_flag.analytics_dashboards.set([analytics_dashboard])
+        usage_dashboard = Dashboard.objects.create(
```

Comment:
minor nit: This already exists, so you don't need to create a new one to use here
```suggestion
        usage_dashboard = existing_flag.usage_dashboard
```